R/ensemble_models.R

```diff
@@ -0,0 +1,677 @@
+#' Ensemble Models
```

Comment:
I see some of the dag patterns being repeated in the code. Is it possible to write it more concisely by creating a function that avoids repetition? As an example:
```r
dplyr::select(-Model_Fit) %>%
        tidyr::unnest(Prediction) %>%
        tidyr::unite(col = "Model_ID", c("Model_Name", "Model_Type", "Recipe_ID"), sep = "--", remove = FALSE) %>%
        dplyr::group_by(Combo_ID, Model_ID, Train_Test_ID) %>%
        dplyr::mutate(Horizon = dplyr::row_number()) %>%
        dplyr::ungroup()
```
is a repeat pattern with a few flavors

Reply:
Discussed offline and decided to not fix based on limitations of foreach which can cause env export issues when running in parallel. 

https://github.com/RevolutionAnalytics/foreach/issues/34